auth/src/main/java/org/jboss/pnc/bacon/auth/model/CacheFile.java

```diff
@@ -100,7 +100,7 @@ private static String generateUsernameMd5(String keycloakUrl, String realm, Stri
 
     private static boolean fileExists(String pathString) {
 
-        Path path = Paths.get(pathString);
+        java.nio.file.Path path = Paths.get(pathString);
```

Comment:
I think nothing now prevents the `java.nio.file.Path` to be imported so you can use only the name `Path` without the package here.